client/src/themes/theme.js

```diff
@@ -4,12 +4,19 @@ export const theme = createMuiTheme({
   typography: {
     fontFamily: '"Poppins"',
     fontSize: 12,
+    color: "red",
```

Comment:
Should this be here? Won't this make all text be red?

Reply:
Also `fontFamily` has nested quotes, should probably be just `"Poppins"`?

Reply:
Added color for testing a thing but forgot to remove it. Nice catch. Fixed it.